Code/Source/URDF/RobotImporter/URDFPrefabMaker.cpp

```diff
@@ -272,4 +269,38 @@ namespace ROS2
         rigidBodyConfiguration.m_inertiaTensor = inertiaMatrix;
         entity->CreateComponent<PhysX::EditorRigidBodyComponent>(rigidBodyConfiguration);
     }
+
+    AZStd::string URDFPrefabMaker::GetAssetPathFromModelPath(std::filesystem::path modelPath)
+    {
+        // It is assumed that the model is in project /Assets folder.
+        // TODO - get asset processor watched directories and support different paths
+        AZStd::vector<AZStd::string> watchedDirectoriesNames = {
+                "Assets"
+        };
+
+        auto azmodelPath = AZStd::string(modelPath.replace_extension("azmodel").string().c_str());
+        AZStd::to_lower(azmodelPath.begin(), azmodelPath.end());
+
+        size_t watchedDirPose = AZStd::string::npos;
+        for (auto watchedDirectory : watchedDirectoriesNames)
+        {
+            AZStd::to_lower(watchedDirectory.begin(), watchedDirectory.end());
+            auto dirToWatch = AZStd::string("/") + watchedDirectory + AZStd::string("/");
```

Comment:
Better to use API for file paths (e.g. filesystem) for portability.

Reply:
No longer applicable - I changed the way how we look for an asset.